content/docs/reference/autocert/readme.md

```diff
@@ -31,7 +31,7 @@ By using autocert, you agree to the [Let's Encrypt Subscriber Agreement](https:/
 
 :::warning
 
-Autocert requires that ports `80`/`443` be accessible from the internet in order to complete a [TLS-ALPN-01 challenge](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01).
+Autocert requires that port `443` be accessible from the internet in order to complete a [TLS-ALPN-01 challenge](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01) or port `80` in order to complete an [HTTP-01 challenge](https://letsencrypt.org/docs/challenge-types/#http-01-challenge) (https://letsencrypt.org/docs/challenge-types/#tls-alpn-01).
```

Comment:
Not a copy review, just noting that the "(https://letsencrypt.org/docs/challenge-types/#tls-alpn-01)" shows up as a parenthese'd weblink